packages/lesswrong/server/migrations/20251014T200342.addManifoldPredictionsOptimizedIndex.ts

```diff
@@ -0,0 +1,10 @@
+import { queueMigrationTask } from "./meta/migrationTaskQueue";
```

Comment:
erm, Idk what's going on with having both this migration and the previous one

Reply:
I too am not sure.  I don't think this migration is doing anything, since neither of the indexes you added are "custom" in the relevant sense?